ts/session/sending/MessageQueue.ts

```diff
@@ -108,20 +95,42 @@ export class MessageQueue implements MessageQueueInterface {
         } else {
           this.events.emit('fail', message, error);
         }
-
-        return result;
       } catch (e) {
         console.warn(
           `Failed to send message to open group: ${message.group.server}`,
           e
         );
         this.events.emit('fail', message, error);
-
-        return false;
       }
+
+      return;
+    }
+
+    let groupId: PubKey | undefined;
+    if (message instanceof ClosedGroupMessage) {
+      groupId = message.groupId;
+    } else if (message instanceof TypingMessage) {
+      groupId = message.groupId;
```

Comment:
we might have a TypingMessage for a non group right?

Reply:
we might, in those cases you would need to send it using `send(device, TypingMessage)`. `sendToGroup` is specifically for sending messages to groups only.